e2e-tests/batch-mode-tests.sh

```diff
@@ -0,0 +1,116 @@
+echo "BATCH MODE TEST"
+​
+echo "STARTING OPENMRS SERVER"
+​
+docker-compose -f openmrs-compose.yaml up -d
+statuscode=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/rest/v1/visittype)
+until [ "$statuscode" -eq 200 ]; do
+      sleep 20s;
+      echo 'WAITING FOR OPENMRS SERVER TO START'
+      statuscode=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/rest/v1/visittype)
+done
+echo 'OPENMRS SERVER STARTED SUCCESSFULLY'
+​
+echo "BUILD THE ANALYTICS PROJECT"
+​
+mvn clean install
+​
+echo "RUN BATCH MODE WITH PARQUET SINK TESTS"
+
+java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar org.openmrs.analytics.FhirEtl --openmrsServerUrl=http://localhost:8099/openmrs --openmrsUserName=admin --openmrsPassword=Admin123  --searchList=Patient,Encounter,Observation --batchSize=20 --targetParallelism=20 --fileParquetPath=/tmp/
+​
+if [ -z "$(ls /tmp/Patient)" ]; then
+  echo 'NO PARQUET FILES FOUND IN /tmp/'
+  exit 0
+fi
+​
+current_path=`pwd`
+
+echo "COPYING PARQUET TOOLS JAR FILE TO ROOT"
+
+cp parquet-tools-1.11.1.jar ~/
+​
+echo "Merge patients, encounter, obs parquet files"
```

Comment:
Do we need to merge each resource? I think you can find the number of records for each resource by simply 
`java -jar ./parquet-tools-1.11.1.jar rowcount /tmp/Patient/` Please feel free to ignore this if the merge operation doesn't take much time

Reply:
I hadn't thought of this, it's actually giving the count as expected. Will replace two merge and rowcount operations with this. Thanks @kimaina